### DISCUSSION TITLE
Date ranges?

### DISCUSSION THREAD

**Discussion by coreybutler** [ncruces/zenity#125] [Q&A] [answered]
Is it possible to select a date range with the calendar dialog?

**Comment by ncruces** [comment, marked as answer]
No, unfortunately.

**Reply by coreybutler** [reply]
Thanks for replying!

I honestly doubt I will pursue it (I can probably get by just asking for a start/end date in separate dialogs), but if I happen to make a new DateRange dialog, would that be something you'd be interested in having merged?

**Reply by ncruces** [reply]
The problem is supporting it in all 3 platforms. And my stance in #11 that the “reference” is mostly `zenity`.

Though it could be argued that the forms dialog might allow this. I'm pretty sure though that on macOS you'll have to use two dialogs.

**Reply by coreybutler** [reply]
If I make this, it would definitely be for all 3 platforms. I'm not familiar enough with all of the intricacies to say I know exactly how it would turn out at this time, and I trust your intuition/experience. I'm doing a fair amount of related work though, so if I have a chance to prototype, I'll start with macOS to see if that is a blocker or if there is a way around it. If I do figure something out, I'll reach out again.

Thanks for your insights, and thanks for this project!
